[2024-12-10T19:12:56.638+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-10T19:12:56.691+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.prepare_data manual__2024-12-10T19:12:44.193340+00:00 [queued]>
[2024-12-10T19:12:56.725+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.prepare_data manual__2024-12-10T19:12:44.193340+00:00 [queued]>
[2024-12-10T19:12:56.727+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-10T19:12:56.821+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): prepare_data> on 2024-12-10 19:12:44.193340+00:00
[2024-12-10T19:12:56.842+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=386) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-10T19:12:56.841+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'ml_pipeline', 'prepare_data', 'manual__2024-12-10T19:12:44.193340+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/mlops.py', '--cfg-path', '/tmp/tmprtuol2bs']
[2024-12-10T19:12:56.846+0000] {standard_task_runner.py:72} INFO - Started process 403 to run task
[2024-12-10T19:12:56.847+0000] {standard_task_runner.py:105} INFO - Job 76: Subtask prepare_data
[2024-12-10T19:12:57.083+0000] {task_command.py:467} INFO - Running <TaskInstance: ml_pipeline.prepare_data manual__2024-12-10T19:12:44.193340+00:00 [running]> on host 15e8047cfc54
[2024-12-10T19:12:57.367+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='prepare_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-10T19:12:44.193340+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-10T19:12:44.193340+00:00'
[2024-12-10T19:12:57.370+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-10T19:12:57.405+0000] {logging_mixin.py:190} INFO - -----Inside prepare_data component----
[2024-12-10T19:12:57.961+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-10T19:12:57.998+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-10T19:12:58.000+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=prepare_data, run_id=manual__2024-12-10T19:12:44.193340+00:00, execution_date=20241210T191244, start_date=20241210T191256, end_date=20241210T191258
[2024-12-10T19:12:58.113+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-10T19:12:58.212+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-10T19:12:58.237+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
